### ISSUE TITLE
Docs: Insecure redirect defaults

### ISSUE THREAD

**Issue by Leksat** [uselagoon/lagoon#3648]
# Describe the bug

The documentation says that the default value for route's `insecure` parameter is `Allow`:
https://github.com/uselagoon/lagoon/blob/d0983ad01f9da22df3c77795e2d805ec9f8ed2c0/docs/concepts-basics/lagoon-yml.md?plain=1#L260-L262

My experience does not confirm this. A route added without the `insecure` parameter redirects to HTTPS.

## To Reproduce

- Add a simple route to `.lagoon.yml`
- => It redirects to HTTPS

## Expected behavior

Docs reflect the actual behavior. Or vice versa 😁

**Comment by shreddedbacon** [comment]
Yes you're right. It looks like the default is [Redirect](https://github.com/uselagoon/build-deploy-tool/blob/dbd45d20e6939de67d298a364d1cf10ee8ec87d1/internal/lagoon/routes.go#L74)
